templates/desktop-menu.twig

```diff
@@ -58,23 +58,5 @@
 				</span>
 		</button>
 	</div>
-	{% if site_languages is not empty %}
-		{% set current_lang = site_languages|filter(i => i.active)|first %}
-		<button class="nav-languages-toggle" type="button"
-				aria-label="{{ __( 'Choose language', 'planet4-master-theme' ) }}"
-				aria-expanded="false"
-				data-bs-toggle="open"
-				data-bs-target="#nav-languages"
-		>{{ current_lang.code|capitalize }}</button>
-		<nav id="nav-languages" class="nav-languages">
-			<ul>
-				{% for key,item in site_languages %}
-					<li class="nav-item {{ item.active ? 'current-language' : '' }}">
-						<a class="nav-link" href="{{ item.url }}">{{ item.native_name }}</a>
-						<span aria-hidden="true">&nbsp;|&nbsp;</span>
-					</li>
-				{% endfor %}
-			</ul>
-		</nav>
-	{% endif %}
```

Comment:
There's some differences between both, but if you compare you'll see it's just formatting, different quote style, and [the aria label](https://github.com/greenpeace/planet4-master-theme/pull/1701/files#diff-cafd9ff56eff72d418df370d7956249ee5ad7a9b90bbea8292dcbac56529b491L31) which was missing on the desktop version.